README.md

```diff
@@ -84,6 +84,9 @@ UMF also provides multithreaded benchmarks that can be enabled by setting both
 `UMF_BUILD_BENCHMARKS` and `UMF_BUILD_BENCHMARKS_MT` CMake
 configuration flags to `ON`. Multithreaded benchmarks require a C++ support.
 
+Scalable Pool benchmark requires libtbb-dev package (libtbbmalloc.so.2) on 
```

Comment:
you could not repeat this information, rather link to the requirements listed in the pool section below

Reply:
done